src/run.py

```diff
@@ -226,14 +226,16 @@ def create_leaflet_map_tiles():
 	)
 
 	# Create map tiles
+      # Create map tiles
 	run_command(
-		'python ' + os.path.join(tilers_tools_directory, 'gdal_tiler.py') + \
-		' --profile=tms' + \
-		' --release' + \
-		' --paletted' + \
-		' --zoom=' + ','.join([str(zoom_level) for zoom_level in range(MIN_ZOOM, MAX_ZOOM + 1)]) + \
-		' --dest-dir=' + intermediate_tiles_directory + \
-		' ' + vrt_file, True
+		'gdal2tiles.py ' + \
+		' --profile=mercator' + \
+		' -x' + \
+		' -r ' + RESAMPLING  + \
```

Comment:
Forgot to commit

`RESAMPLING = 'antialias'

at the top of the file. I'll fix that anon.